src/current/v24.1/cockroachdb-feature-availability.md

```diff
@@ -31,54 +31,60 @@ Any feature made available in a phase prior to GA is provided without any warran
 
-### Custom Metrics Chart page for CockroachDB {{ site.data.products.cloud }} clusters
+{{site.data.alerts.callout_info}}
+Exporting metrics to Azure Monitor is in limited access. Refer to [Exporting metrics to Azure Monitor](#export-metrics-to-azure-monitor).
+{{site.data.alerts.end}}
 
-The [**Custom Metrics Chart** page]({% link cockroachcloud/custom-metrics-chart-page.md %}) for CockroachDB {{ site.data.products.cloud }} clusters allows you to create custom charts showing the time series data for an available metric or combination of metrics.
+### Convert a schema from Oracle or Microsoft SQL Server
 
-### Log SQL Statistics to Datadog
+Using the [Migrations page]({% link cockroachcloud/migrations-page.md %}) to convert a schema from Oracle or Microsoft SQL Server is in preview.
 
-Configure [logging of `sampled_query` events to Datadog]({% link {{ page.version.version }}/log-sql-statistics-to-datadog.md %}) for finer granularity and long-term retention of SQL statistics. The [`sampled_query` events]({% link {{ page.version.version }}/eventlog.md %}#sampled_query) contain common SQL event and execution details for sessions, transactions, and statements.
+### Schema conversion summary report
 
-CockroachDB supports a built-in integration with [Datadog](https://www.datadoghq.com/) which sends query events as logs via the [Datadog HTTP API](https://docs.datadoghq.com/api/latest/logs/). This integration is the recommended path to achieve high throughput data ingestion, which will in turn provide more query events for greater workload observability.
+The [schema-conversion summary report]({% link cockroachcloud/migrations-page.md %}#summary-report) in the Migrations page is in preview. This report displays the results of the schema analysis and provides bulk actions you can apply to update the schema, is in preview.
 
-### Super regions
+### SQL Shell
 
-[Super regions]({% link {{ page.version.version }}/multiregion-overview.md %}#super-regions) allow you to define a set of database regions such that schema objects will have all of their replicas stored _only_ in regions that are members of the super region. The primary use case for super regions is data domiciling.
+The [SQL Shell]({% link cockroachcloud/sql-shell.md %}) in the CockroachDB {{ site.data.products.cloud }} Console is in preview. The SQL enables you to run queries on your CockroachDB {{ site.data.products.cloud }} cluster directly from your browser.
 
-### Export metrics from CockroachDB {{ site.data.products.dedicated }} clusters
+### Restore a CockroachDB {{ site.data.products.dedicated }} cluster from a managed-service backup
+
+[Restoring an entire CockroachDB {{ site.data.products.dedicated }} cluster from a managed-service backup]({% link cockroachcloud/use-managed-service-backups.md %}#restore-a-cluster) is in preview. Managed-service backups are automated backups of CockroachDB Cloud clusters that are stored in Cockroach Labs' cloud storage.
```

Comment:
I think "CockroachDB Cloud" should be "CockroachDB {{ site.data.products.cloud }} "

tbh these YAML vars are so incredibly specific (and LONG) I don't see how they're easier than just writing the names out and doing a search/replace but hey 🤷 😄